sql/moz-fx-data-shared-prod/monitoring_derived/bigquery_table_storage_timeline_daily_v1/metadata.yaml

```diff
@@ -0,0 +1,10 @@
+friendly_name: Bigquery Query Tables Storage Timeline Daily
+description: The daily timeline in table storage changes, partitioned by day.
+owners:
+- wichan@mozilla.com
+labels:
+  incremental: true
+  schedule: daily
+scheduling:
+  dag_name: bqetl_monitoring
+  arguments: ["--date", "{{ ds }}"]
```

Comment:
```suggestion
  arguments: ["--date", "{{ ds }}"]
bigquery:
  time_partitioning:
    field: change_date
    type: day
    require_partition_filter: false
```

Reply:
`require_partition_filter` can also be `true`. Setting it to true is recommended if the data is quite large and we want to prevent users from being able to query a lot of data at once by forcing them to set at date filter